juego.py

```diff
@@ -547,7 +543,7 @@ def save_puntuacionalta(score):
         puntuacionalta = File.readlines()
         File.close()
     p = int(puntuacionalta[0])
-    if not(p > score):
+    if not (p > score):
```

Comment:
Does anyone have any idea why this would not have been written as `p <= score` or indeed just `p < score`?

Reply:
I laughed when I saw this, don't know why though.